addons/crm_livechat/models/mail_channel.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import re
+from odoo import models, _
+
+
+class MailChannel(models.Model):
+    _inherit = 'mail.channel'
+
+    def _define_command_lead(self):
+        return {'help': _('Create a new lead')}
+
+    def _execute_command_lead(self, **kwargs):
+        partner = self.env.user.partner_id
+        msg = _('Something is missing or wrong in command')
+        if kwargs.get('body'):
+            key = kwargs['body'].split()
+            channel_partners = self.env['mail.channel.partner'].search([('partner_id', '!=', partner.id), ('channel_id', '=', self.id)], limit=1)
+            if key[0].lower() == '/lead':
+                if len(key) == 1:
+                    if self.channel_type == 'channel':
+                        msg = _('You are in channel <b>#%s</b>.') % self.name
+                        if self.public == 'private':
+                            msg += _(' This channel is private. People must be invited to join it.')
+                    else:
+                        msg = _('You are in a private conversation with <b>@%s</b>.') % channel_partners.partner_id.name
+                    msg += _('<br>You can create a new lead by typing <b>/lead "lead title"</b>')
+                else:
+                    description = ''
+                    for message in self.channel_message_ids.sorted(key=lambda r: r.id):
+                        name = message.author_id.name or self.anonymous_name
+                        body = re.sub('<[^>]*>', '', message.body)
```

Comment:
@tivisse We must have to escape because in `message.body` message in` <p>` tag (like `<p>message</p>`), Even message may contain link (If we don't escape than it store in description field like `<a href="#">Test Link</a>`).